ci/run-tests.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+set -e
+
+VERSION="${TRAVIS_TAG:-0.0.0}"
+VERSION="${VERSION#v}"
+
+readonly ROOT_DIR="$(dirname "$(dirname "${0}")")"
+cd "${ROOT_DIR}"
+
+function test_fluentd_plugin() {
```

Comment:
Can we add a comment to functions like this that they will change CWD so the caller has to account for that?

Reply:
The function should not change `CWD`.

Reply:
So if I must change `CWD` I should ensure that during going out from the function the `CWD` will be restored?

Reply:
I'd say yes

Reply:
so @perk what is better in your opinion
```
# Didn't test that but should work
{ bundle exec rake && popd; } || { popd && exit 1; }
```

vs

```
  bundle exec rake
  result=$?
  popd
  if [ "$result" -eq "0" ]; then
   exit 1
  fi
```

Reply:
I'd say the latter as it's more straight forward.